
> #### 2017
+ Xingjie Hao, [**Ping Zeng**](https://github.com/biostatpzeng) and Xiang Zhou (2017). 
+ [**Ping Zeng**](https://github.com/biostatpzeng), Xingjie Hao and Xiang Zhou (2017). iMAP 
+ [**Ping Zeng**](https://github.com/biostatpzeng) (2017). Cis-SNPs Set Testing and PrediXcan Analysis for Gene Expression Data using Linear Mixed Model. 
+ [**Ping Zeng**](https://github.com/biostatpzeng) and Xiang Zhou (2017). Non-parametric genetic prediction of complex traits with latent Dirichlet process regression models. **Nature Communications**. 8(1): 456, [Doi: 10.1038/s41467-017-00470-2](https://www.nature.com/articles/s41467-017-00470-2?WT.feed_name=subjects_genetics). [Software](https://github.com/biostatpzeng/DPR)  
+ [**Ping Zeng**](https://github.com/biostatpzeng), Xiang Zhou and Shuiping Huang. Prediction of gene expression with cis-SNPs using mixed models and regularization methods. **BMC Genomics**, 18(368). [Doi: 10.1186/s12864-017-3759-6](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-017-3759-6). [Software](https://github.com/genetics-statistics/GEMMA) 
+ Lorin Crawford, [**Ping Zeng**](https://github.com/biostatpzeng), Sayan Mukherjee and Xiang Zhou (2017). Detecting epistasis with the marginal epistasis test in genetic mapping studies of quantitative traits. **PLoS Genetics**,13(7): e1006869. [Doi: 10.1371/journal.pgen.1006869](http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006869). [Software](https://github.com/lorinanthony/MAPIT)
    
> #### 2015
+ [**Ping Zeng**](https://github.com/biostatpzeng), Yang Zhao, Chen Qian, Liwei Zhang, Ruyang Zhang, Jianwei Gou, Jin Liu, Liya Liu and Feng Chen (2015). Statistical analysis for Genome-Wide Association Study. **Journal of Biomedical Research**, 29(4): 285-297. [Doi: 10.7555/JBR.29.20140007](http://www.jbr-pub.org/ch/reader/view_abstract.aspx?file_no=JBR150403&flag=1).
    
+ [**Ping Zeng**](https://github.com/biostatpzeng) and Ting Wang (2015). Bootstrap restricted likelihood ratio test for the detection of rare variants. **Current Genomics**, 16(3): 194-202. [Doi: 10.2174/1389202916666150304234203](http://www.eurekaselect.com/129127).

> #### 2014
+ [**Ping Zeng**](https://github.com/biostatpzeng), Yang Zhao, Hongliang Li, Ting Wang and Feng Chen (2014). Permutation-based variance component test in generalized linear mixed model with application to multilocus genetic association study. **BMC Medical Research Methodology**, 15(37). [Doi: 10.1186/s12874-015-0030-1](https://bmcmedresmethodol.biomedcentral.com/articles/10.1186/s12874-015-0030-1).

+ [**Ping Zeng**](https://github.com/biostatpzeng), Yang Zhao, Jin Liu, Liya Liu, Liwei Zhang, Ting Wang, Shuiping Huang and Feng Chen (2014). Likelihood ratio tests in rare variant detection for continuous phenotypes. **Annals of Human Genetics**, 78(5): 320-332. [Doi: 10.1111/ahg.12071](http://onlinelibrary.wiley.com/doi/10.1111/ahg.12071/full). [Software](https://github.com/biostatpzeng/LRT)

+ [**Ping Zeng**](https://github.com/biostatpzeng), Yang Zhao, Liwei Zhang, Shuiping Huang and Feng Chen (2014). Rare Variants Detection with Kernel Machine Learning Based on Likelihood Ratio Test. **PLoS ONE**, 9(3): e93355. [Doi: 10.1371/journal.pone.0093355](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0093355).  [Software](https://github.com/biostatpzeng/LRT)

+ [**Ping Zeng**](https://github.com/biostatpzeng), Yongyue Wei, Yang Zhao, Jin Liu, Liya Liu, Ruyang Zhang, Jianwei Gou, Shuiping Huang and Feng Chen (2014). Variable selection approach for zero-inflated count data via adaptive lasso. **Journal of Applied Statistics**, 41(4): 879-894. [Doi:10.1080/02664763.2013.858672](http://www.tandfonline.com/doi/abs/10.1080/02664763.2013.858672). [Software](https://github.com/biostatpzeng/ZIP)
